applications/nrf_desktop/doc/info.rst

```diff
@@ -0,0 +1,42 @@
+.. _info:
```

Comment:
As mentioned, please use `nrf_desktop_*_module` or something like this for all modules.

Reply:
This will be added as separate PR.